gpflow/models/model.py

```diff
@@ -18,6 +18,16 @@
 
 import numpy as np
 import tensorflow as tf
+from tensorflow.python.data.ops import iterator_ops
+
+from .training_interface import (
+    InternalDataTrainingLossMixin,
+    ExternalDataTrainingLossMixin,
```

Comment:
I think these aren't actually needed here ?
```suggestion
```
Or if you want them in here to expose them to people who want to build models, we should add the mcmc one:
```suggestion
    InternalDataTrainingLossMixin,
    ExternalDataTrainingLossMixin,
    MCMCTrainingLossMixin,
```